packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
-## 0.0.1
+## 0.0.13 - 2019-03-15
+* Dark mode, still being polished, is available.  Add ```&theme=dark``` at the end of URI used to open DevTools in the Chrome browser.
+### Memory
+* Added showing GCs on the timeline and leak detection.
+### Timeline
+* Fix bugs when events were received out of order.
 
+## 0.0.1
 - initial (pre-release) release
+### Documentation
+### Debugger
+### Inspector
+### Logging
+### Memory
+### Table
+### Timeline
```

Comment:
These empty headings look a bit weird when the page is rendered, like something is missing. Is there any value in having them? (If they're there as templates for future notes, maybe we could comment them out?)